src/plugins/matching-cards/matching-cards.less

```diff
@@ -91,6 +91,10 @@
   display: flex;
   justify-content: center;
   text-align: center;
+
+  @media @edu-media-phone {
+    font-size: 12px;
```

Comment:
Maybe we could use a variable instead of the hard-coded `px` value here - or - use a percentage like `75%`?

Reply:
Ok, I've used`@edu-font-size-sm` (which was actually not used before).